crm_claim_ref_smartbutton/models/base_model.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import fields, api, models, _
+from lxml import etree
+
+
+base_fields_view_get = models.BaseModel.fields_view_get
```

Comment:
Why the monkeypatch?

Did you know you can inherit BaseModel Normally right?

BTW, due to this has an static folder this code will try to be executed.

Can you try do the feature without monkeypatch?